src/ring_theory/etale.lean

```diff
@@ -300,4 +313,125 @@ formally_etale.iff_unramified_and_smooth.mpr ⟨infer_instance, infer_instance
 
 end base_change
 
+section localization
+
+variables {R S Rₘ Sₘ : Type u} [comm_ring R] [comm_ring S] [comm_ring Rₘ] [comm_ring Sₘ]
+variables (M : submonoid R)
+variables [algebra R S] [algebra R Sₘ] [algebra S Sₘ] [algebra R Rₘ] [algebra Rₘ Sₘ]
+variables [is_scalar_tower R Rₘ Sₘ] [is_scalar_tower R S Sₘ]
+variables [is_localization M Rₘ] [is_localization (M.map (algebra_map R S)) Sₘ]
+local attribute [elab_as_eliminator] ideal.is_nilpotent.induction_on
+
+lemma is_unit.lift_of_nilpotent (x : R) {I : ideal R} (hI : is_nilpotent I)
```

Comment:
What do you think of calling this `is_nilpotent.is_unit_quotient_mk_iff` and turning the statement into an iff?